org.codecoop.kjo.bookletimposer.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
Ideally should be contributed upstream.

Reply:
Its already upstrema but without bookletimposerwrapper.sh so don't know if its usefull

Reply:
AFAIK Exec= is replaced when exported by flatpak.